tests/test_authentication.py

```diff
@@ -16,7 +17,8 @@
 from nose.tools import eq_
 from rest_framework.exceptions import AuthenticationFailed
 
-from hawkrest import HawkAuthenticatedUser, HawkAuthentication
+import hawkrest
+# from hawkrest import HawkAuthenticatedUser, HawkAuthentication
```

Comment:
No longer necessary, since I've now changed the override approach to subclassing.